scipy/linalg/meson.build

```diff
@@ -31,19 +32,19 @@ cython_blas_pxd = cython_linalg[2]
 linalg_init_cython_gen = generator(cython,
   arguments : cython_args,
   output : '@BASENAME@.c',
-  depends : [_cython_tree, __init__py])
+  depends : [_cython_tree, __init__py, __init__pxd])
```

Comment:
This would introduce a small bug, since other submodules (e.g., `cluster`) also depend on this header. Can you revert this change and the two `depends:` entries further down, and instead use:
```meson
_cython_tree += __init__pxd
```
That will keep the `_cython_tree` content unchanged from what it is now.